sdk/src/main/scala/com/horizen/block/SidechainBlockBase.scala

```diff
@@ -37,6 +38,9 @@ abstract class SidechainBlockBase[TX <: Transaction, H <: SidechainBlockHeaderBa
 
   def feePaymentsHash: Array[Byte] = header.feePaymentsHash
 
+  @JsonProperty("size")
+  def size : Long
```

Comment:
to we need to keep 2 separate but equal implementations for account and utxo block?
We may keep it here, no?

```suggestion
  def size: Long = bytes.length
```